src/Parser/RetributionPaladin/Modules/Features/CastEfficiency.js

```diff
@@ -92,7 +92,7 @@ class CastEfficiency extends CoreCastEfficiency {
       spell: SPELLS.BLADE_OF_JUSTICE,
       category: CastEfficiency.SPELL_CATEGORIES.ROTATIONAL,
       getCooldown: haste => null, // 10.5 / (1 + haste)
-      hideWithZeroCasts: true,
+      isActive: combatant => !combatant.hasTalent(SPELLS.DIVINE_HAMMER_TALENT.id),
     },
```

Comment:
you should update the getCooldown to be `` getCooldown: haste => 10.5/(1+haste)`` if you have the formula for it

Reply:
Not accurate due to http://www.wowhead.com/spell=231832/blade-of-wrath. I could add in another blade of justice into the cast efficiency but no one ever takes http://www.wowhead.com/spell=202271/virtues-blade